Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -945,6 +946,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
 
     map.addScreenState(LoginsSettings) { screenState in
         screenState.backAction = navigationControllerBackAction
+        screenState.tap(app.buttons["Settings"], to: SettingsScreen)
```

Comment:
I wondered why this line is needed in the screengraph as well.

Reply:
You may need to tap now in the Settings button to go back to Settings and from there Screengraphs knows how to go to other screens